docs/source/conf.py

```diff
@@ -0,0 +1,61 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import sys
+from pathlib import Path
+
+# patch the Sphinx run so that it can operate directly on the sources
+# see: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#ensuring-the-code-can-be-imported
+module_paths = [
+    Path('..', '..', 'lumigator', 'python', 'mzai', 'sdk').resolve(),
+    Path('..', '..', 'lumigator', 'python', 'mzai', 'schemas').resolve()
+]
+
+for path in module_paths:
+    sys.path.append(str(path))
+
+
+# import the modules that we want to document here to aboid the autodoc error
+# see: https://github.com/pydantic/pydantic/discussions/7763#discussioncomment-8417097
+from sdk import jobs, lm_datasets  # noqa: F401, E402
+
+project = 'Lumigator 🐊'
+copyright = '2024, Mozilla.ai'
+author = 'Dimitris Poulopoulos'
```

Comment:
we'll want to add everyone from the team who's also contributed to the readmes so far, or alternatively "Mozilla AI Engineering" or similar